apps/sms/views/shared/test/unit/localization_helper_test.js

```diff
@@ -20,22 +20,16 @@ suite('LocalizationHelper >', function() {
     return true;
   }
 
-  setup(function() {
-    navigatorMozL10n = navigator.mozL10n;
-    navigator.mozL10n = MockL10n;
+  setup(function(done) {
+    navigatorMozL10n = document.l10n;
+    document.l10n = MockL10n;
 
-    this.sinon.stub(navigator.mozL10n, 'ready');
-    this.sinon.stub(navigator.mozL10n, 'translateFragment');
+    this.sinon.stub(document.l10n, 'translateFragment');
+    this.sinon.stub(document.l10n, 'resolvedLanguages').returns(
+      Promise.resolve([ { code: 'be-BY', dir: 'rtl' } ]));
     this.sinon.stub(window, 'addEventListener');
+    this.sinon.stub(document, 'addEventListener');
 
-    LocalizationHelper.init();
-  });
-
-  teardown(function() {
-    navigator.mozL10n = navigatorMozL10n;
-  });
-
-  test('localizes only attachment iframes on l10n.ready', function() {
     [
       'attachment-container',
       'attachment-container',
```

Comment:
The iframes are now created in the `setup` because `LocalizationHelper.init()` will localize them right away.